### PR TITLE
Allow num0 to float

### PR DESCRIPTION
**Pull request by alanphan22**
Allow num0 to float

### CODE REVIEW

File: calc.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # '1 + 2' -> 3, or '1 - 2' -> -1.
 def compute(expression):
     num0, operator, num1 = expression.split(' ')
-    num0, num1 = int(num0), int(num1)
+    num0, num1 = float(num0), int(num1)
```
Comment (danhphan) [inline calc.py]: Great suggestion, can you also convert num1 to `float`? Thanks

**Review by danhphan — COMMENTED** [review]
Great suggestion, just need a minor update.

**Review by alanphan22 — COMMENTED** [review]
Good job!

**Review by danhphan — APPROVED** [review]
Great!

**Pull request merged**